__tests__/app.test.js

```diff
@@ -180,4 +180,72 @@ describe("app", () => {
         .expect(404)
     });
   });
-}); 
\ No newline at end of file
+}); 
+
+describe("GET /api/articles/:article_id/comments", () => {
+  test("Responds with a status of 200 for the right request.", () => {
+    return request(app).get("/api/articles/1/comments").expect(200);
+  });
+  test("200 - Responds with the requested article_id to the client.", () => {
```

Comment:
Does it really respond with the article id, or responds with the comment that belong to that article with that ID?

Reply:
sorted thanks